src/resources/breaking-changes.md

```diff
@@ -45,46 +45,241 @@ To be notified about future breaking changes, join the [Dart announce][] group.
+
+#### `dart:ffi`
+
+* **Deprecated**: [The `elementAt` pointer arithmetic methods
+  on `Pointer` types are deprecated][54250].
+  Migrate to the `-` and `+` operators instead.
+* **Experimental** **Removed**: The previously deprecated
+  `@FfiNative` annotation has been removed. Usages should be
+  updated to use the `@Native` annotation.
+
+[54250]: https://github.com/dart-lang/sdk/issues/54250
+
+#### `dart:html`
+
+* Instead of using `HttpRequest` directly,
+  it is now recommended to use [`package:http`][].
+
+[`package:http`]: {{site.pub-pkg}}/http
+
+#### `dart:io`
+
+* Instead of using `HttpClient` directly,
+  it is now recommended to use [`package:http`][].
+
+[`package:http`]: {{site.pub-pkg}}/http
+
+#### `dart:js_interop`
+
+* **Experimental**: JS types like `JSAny` have
+  [new compiler-specific representation types][52687].
+* **Experimental**: User-defined `@staticInterop` classes can
+  [no longer implement `JSAny` or `JSObject`][52687].
+  Usages should be migrated to `JSObject.fromInteropObject` or
+  be defined as extension types.
+* **Experimental**: `JSArray` and `JSPromise` now have generic parameters.
+* **Experimental**: Various extension members were moved or renamed.
+  To learn about the updated extensions, reference
+  `JSAnyUtilityExtension` and `JSAnyOperatorExtension`.
+
+[52687]: https://github.com/dart-lang/sdk/issues/52687
+
+#### `dart:typed_data`
+
+* [The unmodifiable view classes for typed data will
+  be deprecated in Dart 3.4][53128].
+  Instead of using the constructors of these classes, use
+  the new `asUnmodifiableView` methods on typed data lists.
+
+[53128]: https://github.com/dart-lang/sdk/issues/53218
+
+#### `dart:nativewrappers`
+
+* **Experimental**: [All native wrapper classes are now marked `base`][51896]
+  so that none of their subtypes can be implemented.
+
+[51896]: https://github.com/dart-lang/sdk/issues/51896
 
 ### Tools
 {: .no_toc}
 
-#### (Dart VM, Pub, Linter, `dart2js`, etc)
-{% endcomment %}
+#### Production JavaScript compiler (dart2js)
+
+* [The `Invocation` that is passed to `noSuchMethod` no longer
+  has a minified `memberName`][54201], even when compiled with `--minify`.
+
+[54201]: https://github.com/dart-lang/sdk/issues/54201
+
+#### Wasm compiler (dart2wasm)
+
+* **Experimental**: [Disallow importing legacy JS interop libraries][54004].
+  Prefer using `dart:js_interop` and `dart:js_interop_unsafe` instead.
 
-## Released in 3.2.0
+[54004]: https://github.com/dart-lang/sdk/issues/54004
+
+#### Analyzer
+
+* **Experimental**: Invalid `dart doc` comment directives are
+  now reported by the analyzer.
+* Due to [improvements in type promotion][54056], the following analyzer
+  diagnostics might trigger on existing code that previously passed analysis:
+
+  * `unnecessary_non_null_assertion`
+  * `unnecessary_cast`
+  * `invalid_null_aware_operator`
+
+[54056]: https://github.com/dart-lang/sdk/issues/54056
+
+#### Linter
```

Comment:
Do these two count as "removed" also?

Reply:
I'm not sure to be honest. I think a bold "Removed" is probably not necessary for these as they don't affect code.

It seemed like **Removed** was more orientated towards language features or APIs that are removed, so users have to change their Dart code.

For now, I don't think these need a lot of attention, so I'll lean towards not adding it.

Let's discuss this page and its categories after things have calmed down a bit :)